### ISSUE TITLE
Error handling for status 200 failure

### ISSUE THREAD

**Issue by hepplerj** [chnm/relec-website#57]
For consideration down the line: do we want to build in error handling in the event the database or API fails to fetch the data?

**Comment by lmullen** [comment]
The dataapi should be pretty well behaved about emitting HTTP errors. There are two areas where it could be better: it should be better about timing out (which currently would not emit an HTTP error, though the client of the API should have its own timeouts) and it should be better about reconnecting to the database if the database restarts (which in practice is extremely rare).

What I've done at various times is, when fetching data that might not be there, write a div with an error message where the SVG would have gone. Something like this (though I'm writing this code without testing it):

```
d3.csv("mydata.csv")
.then(data => {
  doSomething(data);
})
.catch(e => {
  console.log(e)
  d3.select("#target-div")
    .append("p")
    .classed("error", true)
    .text("Sorry, we have a problem on our end.");
});
```

**Comment by hepplerj** [comment]
Resolved in a1e3c8973106b657571be9a371569f1cbe9c0222.
